crates/pop-parachains/src/utils/git.rs

```diff
@@ -46,16 +46,39 @@ impl Git {
 		Ok(())
 	}
 	/// Clone `url` into `target` and degit it
-	pub fn clone_and_degit(url: &str, target: &Path) -> Result<Option<String>> {
+	pub fn clone_and_degit(
+		url: &str,
+		target: &Path,
+		tag_version: Option<String>,
+	) -> Result<Option<String>> {
 		let repo = match Repository::clone(&["https://github.com/", url].concat(), target) {
 			Ok(repo) => repo,
 			Err(_e) => Self::ssh_clone_and_degit(url, target)?,
 		};
 		// fetch tags from remote
 		let release = Self::fetch_latest_tag(&repo);
 
+		if tag_version.is_some() {
+			let tag = tag_version.clone().unwrap();
```

Comment:
Instead of 
```rs
if tag_version.is_some() {
  let tag = tag_version.clone().unwrap();
  ```
  why not use `if let`? 
  ```rs
  if let Some(ref tag) = tag_version {
  
  ```

Reply:
Fixed in Frank's PR